lib/u2f/sign_response.rb

```diff
@@ -4,6 +4,10 @@ class SignResponse
 
     def self.load_from_json(json)
       data = ::JSON.parse(json)
+      if data['clientData'].nil? || data['keyHandle'].nil? || data['signatureData'].nil?
```

Comment:
Why are you doing a `#nil?` check in one place and just checking that the key exists in the other?